docs/guides/guides.md

```diff
@@ -0,0 +1,9 @@
+# Guides
+
+Take a deeper dive into the eWallet. Here are some resources to learn more about how the eWallet works.
+
+- [Usage](/docs/guides/usage.md): Start integrating your eWallet server to your application
+- [API Responsibilities](/docs/guides/api_responsibilities.md): Learn what responsibilities are taken care by the eWallet and what could be built on top of it.
+- [Entities](/docs/guides/entities.md): Learn the important terms used by the eWallet.
```

Comment:
Probably want to switch to `Terminology` based on the description.

Reply:
Terminology probably should cover more than what we have in there. So I renamed terms -> entities instead.